test/check-application

```diff
@@ -333,6 +333,7 @@ class TestFiles(testlib.MachineCase):
         b.wait_in_text("#files-footer-permissions", "rwx r-x r-x")
         m.execute("chmod 700 /home/admin/newdir")
         b.wait_in_text("#files-footer-permissions", "rwx --- ---")
+        b.wait_in_text("#files-footer-owner", "root")
```

Comment:
Let's make this stricter to `wait_text()` -- we don't expect a "root:root" (or even other values) here.